clang/runtime/dpct-rt/include/dpct/kernel.hpp

```diff
@@ -35,6 +35,8 @@ namespace dpct {
 typedef void (*kernel_functor)(sycl::queue &, const sycl::nd_range<3> &,
                                unsigned int, void **, void **);
 
+typedef void (*host_callback)(void *);
```

Comment:
it's make a more common name, which can be used in other context but callback, like host_func